oat-sa/extension-tao-testqti-previewer
# Bugfix/ms 106/readonly text interaction

Ticket: https://oat-sa.atlassian.net/browse/MS-106

At the moment read-only view of item reviewer doesn't allow scrolling in the extended text interaction which means the user can not see the full answer of the test taker at all. This is what is fixed in this PR.

I have added a new plugin in item previewer which is enabled when the previewer is in readOnly mode. The plugin changes the height of the container which holds the test taker's answer to the scroll height of the child container so that the full answer is visible without scrolling.

How to test?

Take delivery with an extended text interaction
Make sure your answer is longer than what can fit inside the extended text interaction's view (content should scroll)
Review the item by going to the Results tab in tao as an admin
The extended text interaction should be able to show the full answer.
